src/test/java/com/sparta/eng82/interfaces/pages/external/LoginPage.java

```diff
@@ -4,13 +4,13 @@
 
 public interface LoginPage extends Page {
 
-    LoginPage enterEmail();
+    LoginPage enterEmail(String email);
 
-    LoginPage enterPassword();
+    LoginPage enterPassword(String password);
 
-    LoginPage enterIncorrectEmail();
+    Page login();
 
-    LoginPage enterIncorrectPassword();
+    boolean loginAttempt(String email, String password);
 
-    Page login();
```

Comment:
No action to click log in anymore?

Reply:
login() is clickLogin() effectively, feel free to edit commit in GitHub if you feel it makes sense

Reply:
Ah, I missed that sorry. Approved.